.github/workflows/hugegraph-llm.yml

```
@@ -0,0 +1,199 @@
+          python -c "from hugegraph_llm.utils.log import log; print('✓ log available')" || echo "✗ log failed"
+          python -c "from retry import retry; print('✓ retry available')" || echo "✗ retry failed"
+        }
+
+    - name: Run unit tests
+      run: |
+        source .venv/bin/activate
+        export SKIP_EXTERNAL_SERVICES=true
+        cd hugegraph-llm
+        echo "=== Running unit tests ==="
+        echo "Current directory: $(pwd)"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)